### DISCUSSION TITLE
nicegui ready to go on port 8081,but I set the the port to 8080

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#3375] [Q&A] [answered]
### Question

I run with `ui.run(port=8080,native=True)`
but the program is ready on port 8081
`NiceGui ready to go on http://localhost:8081, http://169.254.222.119:8081, http://169.254.47.114:8081, http://169.254.48.127:8081, http://169.254.68.226:8081, http://169.254.76.142:8081, http://169.254.9.243:8081, http://169.254.95.245:8081, http://172.30.192.1:8081, http://192.168.0.245:8081, http://192.168.1.17:8081, and http://192.168.1.245:8081`
Also the native doesn't work,there is nothing wrong reported in the terminal.

Previously, there were no issues with the program when `native=True`. I changed the port to 8081, and then I set native to False. The program ran as expected with these new settings. After that, I changed native back to True and the port back to 8080, but the program continued to run on port 8081, and the native setting did not take effect.

Now the app could only run with port 8081,no matter which port I set

**Comment by python-and-novella** [comment]
reboot computer. when `ui.run(port=8080,native=True)`, the output is only `NiceGUI ready to go on http://localhost:8080`

**Reply by v479038280** [reply]
I have already rebooted the computer ,the result is same, It seems the parameter I set with ui.run() doesn't work,not only the native and port ,but also 'show','frameless','fullscreen', the result is always `NiceGui ready to go on http://localhost:8081, http://169.254.222.119:8081, http://169.254.47.114:8081, http://169.254.48.127:8081, http://169.254.68.226:8081, http://169.254.76.142:8081, http://169.254.9.243:8081, http://169.254.95.245:8081, http://172.30.192.1:8081, http://192.168.0.245:8081, http://192.168.1.17:8081, and http://192.168.1.245:8081`

**Reply by v479038280** [reply, marked as answer]
> reboot computer. when `ui.run(port=8080,native=True)`, the output is only `NiceGUI ready to go on http://localhost:8080`

I know the reason ,I test some custom component and that file is with a ui.run(),

**Reply by python-and-novella** [reply]
Before you create a discussion,make sure you hanve a clean env.

**Reply by v479038280** [reply]
Yes, I forget to quote that test line with ui.run()
